### DISCUSSION TITLE
time-to-event models (survival)

### DISCUSSION THREAD

**Discussion by nouranik** [Project-MONAI/MONAI#6769] [Q&A]
Dear MONAI developers and users,
I need to train a time-to-event model for overall survival. is there any way to implement this specific classification model in MONAI? I think the only thing that is needed is a specific loss function to handle censored data.

many thank and looking forward for your helps,
Azadeh

**Comment by KumoLiu** [comment]
Hi @nouranik, thanks for your interest here.
We do not currently have a loss function for survival analysis implemented in MONAI. Welcome to your contribution! Free feel to create an issue or PR.
